src/PointwiseFunctions/GeneralRelativity/WeylElectric.hpp

```diff
@@ -41,6 +41,29 @@ void weyl_electric(
     const tnsr::II<DataType, SpatialDim, Frame>&
         inverse_spatial_metric) noexcept;
 
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes a scalar version of the electric part of the Weyl tensor
+ * in vaccuum.
+ *
+ * \details Computes a scalar version of the electric part of the Weyl tensor
+ * in vaccuum \f$EE\f$ as \f$ EE = E^k_{j} * E^j_{k} \f$ where \f$ E_{ij}\f$ is
```

Comment:
Instead, say `Computes the scalar \f$E_{ij} E^{ij}\f$ from the electric part of the Weyl tensor \f$E_{ij}\f$ and the inverse spatial metric \f$g^{ij}\f$, i.e. \f$E_{ij} = g^{ik}g^{jl}E_{ij}E_{kl}\f$.`